tests/requirements.txt

```diff
@@ -0,0 +1,16 @@
+wheel
+cfgrib
+cftime
+h5py
+fsspec
+numcodecs
+numpy
+ujson
+pandas
+packaging
+zarr
+xarray
+cx_Freeze
+fastparquet
+s3fs
```

Comment:
We don't actually need s3fs or adlfs here; also surprised to see wheel, cx_Freeze and packaging.

Reply:
Without `s3fs`, `fsspec.get_filesystem_class()` raised an error saying 
```
FAILED test.py::test_single - ImportError: Install s3fs to access S3
```

`adlfs` isn't needed, I must have added that by accident. 

`wheel` was added I think because I couldn't `pip install ` the local fsspec-reference-maker package without it I think, but I can't seem to find it in my test logs anymore. It might be fine without it, I'll remove it for now and we'll see what happens

Reply:
You're right I think for `cx_Freeze` and `packaging`, I removed those too. They were added when I used `pipreqs` to determine all the dependencies